### ISSUE TITLE
Cannot upload video above 300mb

### ISSUE THREAD

**Issue by 7aklhz** [parse-community/parse-server#7250]
### New Issue Checklist

- [ x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [ x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x ] I have searched through [existing issues](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).
- [ x] I can reproduce the issue with the [latest version of Parse Server](https://github.com/parse-community/parse-server/releases). 

### Issue Description

I cannot upload videos above 300mb. 

### Actual Outcome

I have changed PARSE_SERVER_MAX_UPLOAD_SIZE to 5000mb and client_max_body_size to 5000m on my NGINX. Now, 
 - When I upload a video of less than 300mb, it works 
 - When I upload a video of 370mb I get "Access to XMLHttpRequest at 'https://xxx/parse/files/myvideo' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource."
 - When I upload a video of 680mb I get "POST https://xxx/parse/files/myvideo 400" and "vue.js:493 Failed to create new xxx, with error code: Invalid file upload."

###  Environment

I've installed parse dashboard, server and mongodb via docker (using caprover)

Server
- Parse Server version: Parse dashboard version : 2.0.5
- Operating system: Docker
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc): OVH

Database
- System (MongoDB or Postgres): MongoDb
- Database version: v4.4.4
- Local or remote host (MongoDB Atlas, mLab, AWS, Azure, Google Cloud, etc): Remote on VPN (using PaaS Caprover)

Client
- SDK (iOS, Android, JavaScript, PHP, Unity, etc): Javascript (vue js from cdn)
- SDK version:

**Comment by mtrezza** [comment]
Thanks for reporting. 

Where should the file be stored and which storage adapter is used?

**Comment by 7aklhz** [comment]
Thank you for getting back to me. The PARSE_SERVER_DATABASE_URI is mongodb://... 

Is that your question ?

**Comment by mtrezza** [comment]
Are you not using a storage adapter like S3, but trying to store a 300MB file directly in the DB?
